shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -375,24 +377,24 @@ export function isPlainObject(value: any): boolean {
     let result: boolean = false;
 
     if (value && typeof value === "object") {
-        let proto = _getObjProto(value);
+            // Inlining _objGetPrototypeOf for performance to avoid an additional function call
```

Comment:
Lots of perf improvements, reduces the runtime (in the perf tests) locally from over 200ms for 100000 iterations to < 50ms (30ish on Chromium Edge and 40is on Firefox)